server-side/src/app.module.ts

```diff
@@ -0,0 +1,17 @@
+import { Module } from '@nestjs/common';
+import { AppController } from './app.controller';
+import { AppService } from './app.service';
+import { AuthModule } from './auth/auth.module';
+import { MongooseModule } from '@nestjs/mongoose';
+import { JwtModule } from '@nestjs/jwt';
+
+@Module({
+  imports: [
+    AuthModule,
+    MongooseModule.forRoot('mongodb://localhost:27017/translator'),
+    JwtModule.register({ secret: '123' }),
```

Comment:
In real projects, make sure you put sensitive data in a .env file and get it from there